repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/identityv3/atom/wauthenticatedfeed/openstack-identity-v3.cfg.xml

```diff
@@ -7,6 +7,7 @@
                                 uri="http://localhost:${identityPort}"
                                 xmlns="http://docs.openrepose.org/repose/openstack-identity-v3/v1.0"/>
     <cache>
+        <timeouts group="6000" token="6000"/>
```

Comment:
I'm concerned this was neccessary

Reply:
A justified concern. It seems to have been necessary because of a bug that was introduced. We seem to have switched over from milliseconds to seconds as the default value for these timeouts, but the code was not updated to support that switch.

We will ultimately use seconds, and the code will support them as such.

Reply:
Looks like this line was removed, so I'm resolving the convo.